pai-management/paictl.py

```diff
@@ -163,26 +165,38 @@ def cluster_object_model_generate_service(config_path):
 
 
 
-def hadoop_ai_build(os_type = "ubuntu16.04"):
+def pai_build():
 
     parser = argparse.ArgumentParser()
     parser.add_argument('-p', '--path', required=True, help="The path of your configuration directory.")
+    parser.add_argument('-n', '--imagename', default='all', help="Build and push the target image to the registry")
+    parser.add_argument('-a', '--action', default=None, help="Only build or push" )
```

Comment:
I suggest paictl using 2 level command format:
paictl <first-level-module> [first-level-parameters] <second-level-command> [second-level-parameters]

This will very like Linux "ip" command:
core@paigcr-a-gpu-1033:~$ ip
Usage: ip [ OPTIONS ] OBJECT { COMMAND | help }
       ip [ -force ] -batch filename

By this way, we force user to write command in logically ordered way to avoid some mis ordered usage such as "paictl image --image-name=xxx --config-path=xxx -a push".

Reply:
resolved